.travis.yml

```diff
@@ -5,9 +5,13 @@ go:
 
 services:
   - docker
+  - postgresql
 
 sudo: false
 
+before_script:
+  - psql -c 'create database docker_test;' -U postgres
+
```

Comment:
You'd still need to specify a db to connect to for psql - unless there is
some sort of default db. Anyways - this wont be running on windows :)

On Fri, Jun 10, 2016 at 1:05 PM, davidamitchell <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/balanceit/docker_test/pull/1#discussion_r66547878>:
>
> >
> >  sudo: false
> >
> > +before_script:
> > +  - psql -c 'create database docker_test;' -U postgres
> > +
>
> sure for the createdb command the last option has be to be db name, but
> for psql -c you are just sending a sql query so it doesn't matter (i
> don't think, don't have a windows machine which i care to test it on)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/balanceit/docker_test/pull/1/files/a4f54991de4a78eebdffb357d01a6bef446e85e3#r66547878>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAkQSrEBvt91qlgYe_H6174n2Y_Qw0Wbks5qKLhDgaJpZM4IyY_U>
> .
>